### ISSUE TITLE
Error logs when leaving RTP users

### ISSUE THREAD

**Issue by BerdaSantos** [mconf/bbb-webrtc-sfu#141]
**Endpoint(s) description :**
WebRTC users (firefox/chrome) and RTP Users

**Media format :**
Video H.264

**version/hash:**
0a8c4c2

**Problem description / Steps to reproduce:**
The application reproduces the error log. / RTP user enters and leaves the room.

**Expected Behavior:**
Don't show error logs.

**Extra INFO**
LOG OUTPUT:
```
2019-10-21T20:41:25.014Z - error: [mcs-router] Reject operation leave with type=error, code=undefined, message=user.leave(...).then(...).catch(...).finally is not a function, details=undefined, operation=leave

2019-10-21T20:41:25.014Z - info: [ScreenshareManager] Stopping session 70123

2019-10-21T20:41:25.015Z - error: [mcs-api] Reject operation leave with type=error, code=undefined, message=user.leave(...).then(...).catch(...).finally is not a function, details=undefined, stack=undefined, operation=leave
```

**Comment by mariogasparoni** [comment]
There's a patch already being done to solve this issue. I'll submit in a couple minutes

**Comment by prlanzarin** [comment]
Addressed in #142 .
